ramlfications/loader.py

```diff
@@ -1,25 +1,21 @@
 # -*- coding: utf-8 -*-
 # Copyright (c) 2015 Spotify AB
 
-try:
-    from collections import OrderedDict
-except ImportError:  # pragma: no cover
-    from ordereddict import OrderedDict
-
 import os
-
 import jsonref
 import yaml
 
 from six import string_types
 
 from .errors import LoadRAMLError
+from .utils.common import OrderedDict
 
 
 __all__ = ["RAMLLoader"]
 
 
 RAMLHEADER = "#%RAML "
+SUPPORTED_FRAGMENT_TYPES = ("DataType",)
```

Comment:
I'm assuming this is for stuff like `#%RAML 1.0 Library` and `#%RAML 1.0 DataType` etc. in file headers?